src/rebdhuhn/graph_utils.py

```diff
@@ -40,6 +40,10 @@ def _mark_last_common_ancestors(graph: DiGraph) -> None:
     Each node which is such an ancestor will contain the information of which nodes it is the last common ancestor.
     It is stored in the dict field `COMMON_ANCESTOR_FIELD` as a list.
     """
+    if len(graph.nodes) > 90:
+        raise GraphTooComplexForPlantumlError(
+            message=f"Graph is too large to determine the last common ancestors." f"Number of Nodes: {len(graph.nodes)}"
+        )
```

Comment:
```suggestion
    if len(graph.nodes) > 90:
        raise GraphTooComplexForPlantumlError(
            message=f"Graph is too large to determine the last common ancestors. Number of Nodes: {len(graph.nodes)}"
        )
```
Als ob, ist ja räudig.